src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -164,6 +164,7 @@ def serialize(self, obj, attrs, user, **kwargs):
             "dateModified": obj.date_modified,
             "dateCreated": obj.date_added,
             "createdBy": attrs.get("created_by", None),
+            "snooze": False,
```

Comment:
It looks like for alert rules we didn't add the snooze stuff the serializer, but just added the extra stuff later on: https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/project_rule_details.py#L88-L99

Reply:
Hmm which test is this?

Reply:
https://github.com/getsentry/sentry/blob/master/tests/sentry/incidents/endpoints/test_project_alert_rule_details.py#L124

Reply:
Oh I see what you mean ... you can leave it but I tried that test w/o this line in the serializer and it passes, I might be missing something though.

Reply:
i think it works now because it isn't setting snooze to False anywhere else, in the other version, we set snooze to false as an else in the endpoint. if you add that in, then it wouldn't match the serailizer.